packages/node_modules/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -407,7 +407,7 @@ describe('plugin-mercury', () => {
         delete webex.config.defaultMercuryOptions;
       });
 
-      it('connects to Mercury using proxy agent', () => {
```

Comment:
Skipping for now. Need to create a JIRA to fix this test.

Reply:
Is this broken on master?

Reply:
Yes

Reply:
Could you create a JIRA bug for this failure and add the ticket number as a TODO comment?